refactor: new CLI commands documentation (#413)
NodeSecure/cli: docs/cli/open.md

## 📝 Command `open`

> The `open` command is used to open a security report in a local HTTP server. This allows for easy access and review of the security report through a web browser.

Reviewer: ```suggestion
The `open` command reads a specified JSON payload and starts a local HTTP server. This allows you to explore dependencies, their metrics, and potential threats directly in your web browser.
```

## 📜 Syntax

```bash

> nsecure open

Reviewer: ```suggestion
nsecure open [json]
```

```

> 

Reviewer: ```suggestion

> [!NOTE]
> If the `[json]` property is omitted, the command will default to searching for a `nsecure-result.json` file in the current working directory.

```
